docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -41,12 +41,14 @@ $ kubectl get namespaces
 NAME          STATUS    AGE
 default       Active    1d
 kube-system   Active    1d
+kube-public   Active    1d
 ```
 
-Kubernetes starts with two initial namespaces:
+Kubernetes starts with three initial namespaces:
 
    * `default` The default namespace for objects with no other namespace
    * `kube-system` The namespace for objects created by the Kubernetes system
+   * `kube-public` The namespace is created automatically and readable by all users (including those not authenticated).
```

Comment:
... This namespace is mostly reserved for system usage, in case some resource should be readable publicly. Further, the public aspect of this namespace is only a convention, not a requirement.